client/src/questions-n-answers/qNa.jsx

```diff
@@ -137,15 +201,73 @@ class QuestionsNAnswers extends React.Component {
           .then(data => {
             questions(this.props.product_id)
               .then(newData => {
-                let filtered =this.filterAnswersNQuestions(newData.data)
-                console.log(filtered)
-                this.setState({
-                  answers: filtered[1]
+
+                let sortedData = this.filterAnswersNQuestions(newData.data)
+                let showButton = this.helper().showMoreAnsweredQuestions(sortedData)
```

Comment:
I'm curious as to why helper is invoked? I would have figured showMoreAnsweredQuestion was a method of a helper class, but is helper instead a function that needs to be run and SMAQ is chained to get the result of calling it?